examples/plotting/file/data_models.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+
+from bokeh.plotting import figure
+from bokeh.models import ColumnDataSource, CustomJS
+from bokeh.io import save
+
+from bokeh.model import Model
+from bokeh.core.properties import Float
+
+class Params(Model):
+    __data_model__ = True
```

Comment:
This all looks really good, making the UX a little more streamlined is my only real comment. Presumably we can make this simpler for users e.g. with something like 
```python
Class DataModel(Model):
     __data_model__ = True
```
That users can inherit from instead of `Model`

Reply:
I added `DataModel`. Setting `__data_model__` explicitly is still useful, because one can extend a proper model with additional fields, thus can't inherit from `DataModel`.